ONSdigital/takeon-business-layer
# Feature/spp 827 validation date adj

# Description



Using Adjusted value for RSI validation. BMI validation will continue to run on contributor response data.
https://collaborate2.ons.gov.uk/jira/browse/SPP-827: Use Date Adjusted value for validation if applicable
# How to test?




Run Validation from UI ( https://dc-ui-spp827.es-take-on.aws.onsdigital.uk/ ) . Check the CloudWatch logs in spp-es-takeon-validation-spp827-wrangler lambda. It will show which response value is used for which question and accordingly passed to the formula that will be evaluated for validation. Tested Data e.g. Reference:12000613746 Survey:023 Period:201903
# Links


https://github.com/ONSdigital/takeon-validation/pull/37
